netclient/gui/components/views/networks.go

```diff
@@ -49,6 +50,24 @@ func GetNetworksView(networks []string) fyne.CanvasObject {
 				leave(*network)
 			}, components.Danger_color),
 		)
+		cfg, err := config.ReadConfig(*network)
+		if err != nil {
+			fmt.Println(err)
+		}
+		if cfg.Node.Connected == "yes" {
+			grid.Add(
+				components.ColoredIconButton("connection", theme.CheckButtonCheckedIcon(), func() {
+					disconnect(*network)
+				}, components.Gravitl_color),
+			)
+		} else {
+			grid.Add(
+				components.ColoredIconButton("connection", theme.CheckButtonIcon(), func() {
```

Comment:
ColoredIconButton("connect"

would be better to call it "connect" so if you click it it will connect

Reply:
Okay